ldi-core/ldes-client/tree-node-supplier/pom.xml

```diff
@@ -69,6 +69,10 @@
             <artifactId>HikariCP</artifactId>
             <version>${hikari.version}</version>
         </dependency>
+        <dependency>
+            <groupId>be.vlaanderen.informatievlaanderen.ldes.ldi</groupId>
+            <artifactId>ldi-common</artifactId>
+        </dependency>
```

Comment:
Depedency is already already added at lines 14-18

Reply:
was a rebase thing that i got from PJ's commit .. will remove thanks